File: ShowMyName/app/src/main/java/com/christiantisby/showmyname/MainActivity.java
package com.christiantisby.showmyname;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    private Button showButton;
    private TextView textView;
    private EditText editText;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        showButton = findViewById(R.id.myButton);
        textView = findViewById(R.id.myTextView);
        editText = findViewById(R.id.editText);

        showButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                String name = editText.getText().toString();
                if (name.isEmpty()) {
                    textView.setText("Who are you???");
                } else {
                    textView.setText("Hello, " + name);
                }

            }
        });



    }
}

File: PixelDensity/app/src/main/java/com/christiantisby/pixeldensity/MainActivity.kt
package com.christiantisby.pixeldensity

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        supportActionBar!!.setDisplayShowHomeEnabled(true)
        supportActionBar!!.setLogo(R.mipmap.ic_launcher)
        supportActionBar!!.setDisplayUseLogoEnabled(true)
        setContentView(R.layout.activity_main)
    }
}